cobl2/templates/dataset/detail_html.mako

```diff
@@ -8,11 +9,44 @@
         </p>
     </div>
 </%def>
+</%doc>
 
-<h2>Welcome to CoBL-IE</h2>
+<h2>Welcome to IE-CoR</h2>
 
-<h3>Cognacy in Basic Lexicon for the Indo-European language family</h3>
+<span style='font-size:120%'>a new breed of language databases on <strong>Co</strong>gnate <strong>R</strong>elationships in core vocabulary</em></span>
 
 <p class="lead">
-    One basic way of assessing how closely certain languages are related to each other is through ‘cognacy’, i.e. to what extent they still share words that go back to the same origin or not. English ‘salt’, German ‘Salz’ and French ‘sel’, for instance, are all cognates, i.e. related words that all go back to the same original source word, in those languages’ single common ancestor language (‘Proto-Indo-European’). On the other hand, ‘black’, ‘schwarz’ and ‘noir’ all go back to different source words, because of shifts in which word is used to represent which meaning.
+
+<h4>What are ‘<strong>Co</strong>gnate <strong>R</strong>elationships’?</h4>
+
+<p>One basic way of assessing how closely certain languages are related to each other is through ‘cognacy’, i.e. to what extent they still share words that go back to the same origin or not.  English ‘salt’, German ‘Salz’ and French ‘sel’, for instance, are all cognates, i.e. related words that all go back to the same original source word, in those languages’ single common ancestor language (‘Proto-Indo-European’).  On the other hand, ‘black’, ‘schwarz’ and ‘noir’ all go back to different source words, because of shifts in which word is used to represent which meaning.</p>
+
+<h4>The CoR Database System</h4>
+
+<p>CoR is a new database structure for exploring the <strong>Co</strong>gnate <strong>R</strong>elationships between languages in their basic or &#8216;core&#8217; vocabulary.  The core vocabulary covered is a reference list of 170 basic and common word meanings, such as ‘one’, ‘water’, ‘black’, and so on.  CoR uses a new 170 meaning list, based on a combination and adaptation of three lists already widely used in linguistics:  the Swadesh lists (100 and 200), and the Leipzig-Jakarta list (100).  This combination has been heavily revised to optimise it for consistency in both lexeme determination, and cognacy determination within Indo-European.  All 170 meanings have also been precisely (re)defined. </p>
+
+<p>CoR is tailored for qualitative as well as quantitative research purposes, and includes data-exploration websites to search the rich linguistic data covered:  cognate sets, orthography, morphology, phonemic and IPA phonetic transcriptions, and links to further sources.  </p>
```

Comment:
I'd leave the explanation of the database system to the demo app of that system, i.e. remove the "The CoR Database System" section here.

Reply:
but in this section is explained which meanings are taken and why and how - the last paragraph of this section could be removed maybe - but I think, the section header name could only be changed into `The CoR data` or similar

Reply:
Ah, ok. Maybe we just keep the sentence explaining this? The 170 item list isn't fixed in the database system anyway, right? So it shouldn't be described as a defining feature of it?

Reply:
ok